bpfman/src/bpf.rs

```diff
@@ -658,6 +664,52 @@ impl BpfManager {
 
                 Ok(id)
             }
+            Program::Fentry(ref mut program) => {
+                let fn_name = program.get_fn_name()?;
+                let btf = Btf::from_sys_fs()?;
+                let fentry: &mut FEntry = raw_program.try_into()?;
+                fentry
+                    .load(&fn_name, &btf)
+                    .map_err(BpfmanError::BpfProgramError)?;
+                program.get_data_mut().set_kernel_info(&fentry.info()?)?;
+
+                let id = program.data.get_id()?;
+                let link_id = fentry.attach()?;
+                let owned_link: FEntryLink = fentry.take_link(link_id)?;
+                let fd_link: FdLink = owned_link.into();
+                fd_link
+                    .pin(format!("{RTDIR_FS}/prog_{}_link", id))
```

Comment:
```suggestion
                    .pin(format!("{RTDIR_FS}/prog_{id}_link"))
```

Reply:
same below

Reply:
Done.
There are multiple places that the variable is in the string, why is this method preferred?

Reply:
If it's already in the right format it's just a bit less code :) but really it's just a style thing